django_spanner/introspection.py

```diff
@@ -38,7 +38,7 @@ class DatabaseIntrospection(BaseDatabaseIntrospection):
             FROM
                 information_schema.tables AS t
             WHERE
-                t.table_catalog = '' and t.table_schema = ''
+                t.table_catalog = '' and t.table_schema = '{schema_name}'
```

Comment:
Is the `if USE_EMULATOR` statement still needed?

Reply:
Removed as the mentioned bug is closed and support has been added to emulator